components/status/StatusActionsMore.vue

```diff
@@ -42,10 +48,9 @@ function getPermalinkUrl(status: mastodon.v1.Status) {
   return null
 }
 
-async function copyLink(status: mastodon.v1.Status) {
-  const url = getPermalinkUrl(status)
-  if (url)
-    await clipboard.copy(url)
+async function copyLink() {
+  if (statusRoute)
+    await clipboard.copy(statusRoute)
```

Comment:
I honestly don't know enough about Nuxt to understand why we were ever passing the status link in to this function since we have the status info so PLEASE check my assumption here.

Reply:
yep i agree, seems unnecessary. for consistency we should remove it from `copyOriginalLink` below as well.

Reply:
Done!